docker/nginx/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM nginx:latest as gewisdb_nginx
+FROM nginx:stable-alpine as gewisdb_nginx
+
+RUN adduser -D -H -u 1000 -s /bin/bash www-data -G www-data
```

Comment:
You mention bash is no longer available. Yet, we use it here?

Also, there are several references to bash in other files that we should probably replace.
![afbeelding](https://user-images.githubusercontent.com/32361020/183240410-435195c5-26da-48ae-9553-a0d5e794a786.png)

Reply:
Missed these, will change them!